secretmanager/delete_secret_with_etag.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package secretmanager
+
+// [START secretmanager_delete_secret_with_etag]
+import (
+	"context"
+	"fmt"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	secretmanagerpb "google.golang.org/genproto/googleapis/cloud/secretmanager/v1"
+)
+
+// deleteSecretWithEtag deletes the secret with the given name and all of its versions.
+func deleteSecretWithEtag(name, etag string) error {
+	// name := "projects/my-project/secrets/my-secret"
+	// etag := "\"123\""
```

Comment:
Should this value actually be a quoted string? If so maybe use the a ` character quotation to avoid having to escape.

Reply:
Yes, it is supposed to be quoted as per https://google.aip.dev/154. I'll fix the notation. Let me also squash the commits, not sure why the ops from commandline don't show up in the ui.

Reply:
We always squash on merge so don't worry about the history 😄

Reply:
Fixed now.